src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -134,4 +135,9 @@ public function getExtendedType()
     {
         return 'Symfony\Component\Form\Extension\Core\Type\FormType';
```

Comment:
What do you think about `return self::extendsType();` to make clearer the migration path?